src/custom/components/swap/TradePrice/index.tsx

```diff
@@ -1,25 +1,8 @@
-import React, { useMemo } from 'react'
+import React from 'react'
 import TradePriceMod, { TradePriceProps } from './TradePriceMod'
-import { useHigherUSDValue /*  useUSDCValue */ } from 'hooks/useUSDCPrice'
-import { formatSmart } from 'utils/format'
-import { tryParseAmount } from 'state/swap/hooks'
-import { FIAT_PRECISION } from 'constants/index'
 
 export * from './TradePriceMod'
 
-export default function TradePrice(props: Omit<TradePriceProps, 'fiatValue'>) {
-  const { price, showInverted } = props
-
-  const priceSide = useMemo(
-    () =>
-      !showInverted
-        ? tryParseAmount(price.invert().toFixed(price.baseCurrency.decimals), price.baseCurrency)
-        : tryParseAmount(price.toFixed(price.quoteCurrency.decimals), price.quoteCurrency),
-    [price, showInverted]
-  )
-  // const amount = useUSDCValue(priceSide)
-  const amount = useHigherUSDValue(priceSide)
-  const fiatValueFormatted = formatSmart(amount, FIAT_PRECISION)
-
-  return <TradePriceMod {...props} fiatValue={fiatValueFormatted} />
+export default function TradePrice(props: TradePriceProps) {
+  return <TradePriceMod {...props} />
```

Comment:
Why not exporting this directly if u don't modify anything?

Reply:
artifact, i am removing this. before i had logic in here but moved it away, good catch